ios/Classes/CamerawesomePlugin.m

```diff
@@ -107,10 +107,6 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
         [self _handleSetup:call result:result];
     } else if ([@"checkPermissions" isEqualToString:call.method]) {
         [self _handleCheckPermissions:call result:result];
```

Comment:
Do we need to remove this code as well?

Reply:
Permission check is still in there, requesting permissions is removed.